Jenkinsfile

```diff
@@ -26,70 +26,52 @@ def sendEmailNotifications() {
     )
 }
 
-def BuildVersionX(isPublishNeeded, version) {
+def BuildVersionX(version) {
     def gradleCommand = "./gradlew --info --stacktrace "
-    def implProject = ":apix-impl:apix-impl-${version}"
-
-    // Unit tests
-    sh "${gradleCommand} ${implProject}:test"
-
-    // Integration tests
+    sh "${gradleCommand} :apix-impl:apix-impl-${version}:test"
```

Comment:
Doesn't hurt to clarify the difference between `:test` and `:integrationTest` again?